### DISCUSSION TITLE
希望能增加「灋」字和「䘙」字的收錄，以及建議增加「替換優先字」的功能

### DISCUSSION THREAD

**Discussion by newnamegaming** [rime/home#1199] [Ideas]
因爲「灋」是「法」的古字，「䘙」是「衛」的古字。
替換優先字的實現方法是：
採用另建csv格式檔案儲存，但是csv檔案名要爲對應的輸入法名稱，逗號左邊爲被替換位置的字，逗號右邊爲替換上去的字，即：
`A,B`
A爲原本的優先字，B爲原本的非優先字，該行內容收錄進去後，變成B爲優先字，A爲非優先字
如果A和B在對應輸入法中不是共同的碼位，則這一行會被ignore掉。

**Comment by LEOYoon-Tsaw** [comment]
用OpenCC就行了，可以用txt做其配置檔

**Reply by newnamegaming** [reply]
> 用OpenCC就行了，可以用txt做其配置檔

怎麼做配置檔？內有一個「to_tw_variants.txt」好像試過了也不管用，因爲我是無聲調粵拼方案。

**Reply by LEOYoon-Tsaw** [reply]
跟你什麼方案沒有干係，只要是漢字就能用。網上例子應該不少，找不到的話我可以幫你找一個

**Reply by newnamegaming** [reply]
> 跟你什麼方案沒有干係，只要是漢字就能用。網上例子應該不少，找不到的話我可以幫你找一個

你知道怎麼設嗎？
